Makefile

```diff
@@ -232,6 +232,11 @@ central-build-nodeps:
 	@echo "+ $@"
 	$(GOBUILD) central
 
+.PHONY: config-controller-gen
```

Comment:
this should run with make gen on the top level as well so we will need to add this to the make generate target

Reply:
Can you explain what you mean by `make gen`? I tried both `make gen` and `make generate` at the top level, and it didn't work.  Are you suggesting we add that as a top-level make target?

Reply:
I meant the top level repo make generated-srcs target should ensure controller stuff is also generated. So while having a separate target works, the ` go-generated-srcs` target should invoke this one too. Also there are clean targets for generated srcs which will need the corresponding change.

Reply:
Check out b04596f6d9ce10a7641a98ea8905d0c2b8bb9d7f and let me know if that's what you were thinking.